File: api/code-samples/python/klaytnDebug/logging/backtraceAt.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

location = "server.go:443"

sdk = OpenSDK(host)
debug_response = sdk.debug.backtrace_at(location)

print(json.loads(debug_response.response.data))

File: web3rpc/rpc-specs/api/code-samples/javascript/debug/logging/backtraceAt.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const location = "server.go:443";

    sdk.debug.backtraceAt(location, {}, (err, data, response) => {
        console.log(data);
    });
}
)()